config/src/config/secure_config.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::{Deserialize, Serialize};
+use std::path::PathBuf;
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct SecureConfig {}
```

Comment:
Where does secure backend go? I would just call this SecureBackend and have the enum be the top level element and have it implement some reasonable default like it currently does.

Can we separate this from your stack? I'd like to build on top of it rather than parallel.

Reply:
Thanks David. As discussed offline, we'll have a SecureBackend in SafetyRulesConfig and KeyManagerConfig for now and then do a clean up of this stuff in a future PR!

Reply:
why are we defining SecureConfig in this commit?

Reply:
Commit bleed -- thanks for pointing it out!